projects/plugins/jetpack/class.jetpack.php

```diff
@@ -7398,94 +7398,78 @@ public static function is_active_and_not_offline_mode() {
-			'options'           => array(
-				array(
-					'type' => 'search',
-					'slug' => 'jetpack-search',
-					'key'  => 'jetpack_search',
-					'name' => __( 'Search', 'jetpack' ),
-				),
+		$products['scan'] = array(
+			'title'             => __( 'Jetpack Scan', 'jetpack' ),
+			'slug'              => 'jetpack_scan',
+			'description'       => __( 'Automatic scanning and one-click fixes keep your site one step ahead of security threats and malware.', 'jetpack' ),
+			'show_promotion'    => true,
+			'discount_percent'  => 40,
+			'included_in_plans' => array( 'security' ),
+			'features'          => array(
+				_x( 'Automated daily scanning', 'Scan Product Feature', 'jetpack' ),
+				_x( 'One-click fixes for most issues', 'Scan Product Feature', 'jetpack' ),
+				_x( 'Instant email notifications', 'Scan Product Feature', 'jetpack' ),
+			),
+		);
+
+		$products['search'] = array(
+			'title'             => __( 'Jetpack Site Search', 'jetpack' ),
+			'slug'              => 'jetpack_search',
+			'description'       => __( 'Help your site visitors find answers instantly so they keep reading and buying. Great for sites with a lot of content.', 'jetpack' ),
+			'show_promotion'    => true,
+			'discount_percent'  => 40,
+			'included_in_plans' => array(),
+			'features'          => array(
+				_x( 'Instant search and indexing', 'Search Product Feature', 'jetpack' ),
+				_x( 'Powerful filtering', 'Search Product Feature', 'jetpack' ),
+				_x( 'Supports 29 languages', 'Search Product Feature', 'jetpack' ),
+				_x( 'Spelling correction', 'Search Product Feature', 'jetpack' ),
 			),
-			'tears'             => array(),
-			'default_option'    => 'search',
-			'show_promotion'    => false,
-			'included_in_plans' => array( 'search-plan' ),
 		);
 
-		$products[] = array(
-			'key'               => 'anti-spam',
+		$products['akismet'] = array(
 			'title'             => __( 'Jetpack Anti-Spam', 'jetpack' ),
-			'short_description' => __( 'Automatically clear spam from comments and forms. Save time, get more responses, give your visitors a better experience – all without lifting a finger.', 'jetpack' ),
-			'learn_more'        => __( 'Learn More', 'jetpack' ),
-			'description'       => __( 'Automatically clear spam from comments and forms. Save time, get more responses, give your visitors a better experience – all without lifting a finger.', 'jetpack' ),
-			'options'           => array(
-				array(
-					'type' => 'anti-spam',
-					'slug' => 'jetpack-anti-spam',
-					'key'  => 'jetpack_anti_spam',
-					'name' => __( 'Anti-Spam', 'jetpack' ),
-				),
+			'slug'              => 'jetpack_anti_spam',
+			'description'       => __( 'Save time and get better responses by automatically blocking spam from your comments and forms.', 'jetpack' ),
+			'show_promotion'    => true,
+			'discount_percent'  => 40,
+			'included_in_plans' => array( 'security' ),
```

Comment:
Shouldn't we leave the legacy plans in there?

Reply:
> Shouldn't we leave the legacy plans in there?

For the new use-case (product description "pages") we're, for now, only planning on upselling the security bundle and are not planning on upselling legacy bundles which means we don't need the data. We can still leave it there but we would leave it without any particular use-case which IMO can be confusing for other developers (e.g.: are there related legacy plan code elsewhere etc).

I'm up to both solutions, so if you feel strongly about one over the other, please let me know and I'll update if it necessary 😄